Jenkinsfile

```diff
@@ -13,6 +13,9 @@
 // The the built hypothesis/viahtml Docker image.
 def img
 
+// The app we are passing to deployApp
+def app = "$env.JOB_NAME"
```

Comment:
Might be nice to give an example and explain what this is doing? A bit opaque from this alone.

e.g.

```
// Unlike other apps ViaHTML is deployed to more than one location, so...
// blah blah
```